addons/base_automation/models/base_automation.py

```diff
@@ -26,6 +26,19 @@
 }
 
 
+class AutomationFailureWizard(models.TransientModel):
+    _name = 'base.automation.failure.wizard'
+    _description = 'Automated action failure wizard'
+
+    def deactivate(self):
+        failures = self.env['base.automation'].browse(self._context.get('active_ids'))
```

Comment:
this commit was not intended to be back-portable, I think it's dangerous to simply allow users to disable automated actions which might render their future data corrupted without displaying a big warning message...

Perhaps for stable we could simply add a button which will open the backend form view of the `base.automation`

OK for your first comment

Reply:
> this commit was not intended to be back-portable, I think it's dangerous to simply allow users to disable automated actions which might render their future data corrupted without displaying a big warning message...

Of course, and the feature itself is very risky for this reason. What I meant is that the "big warning message"  would have to be implemented client-side in JS, not with a server-side wizard, if you want to be able to backport it. 
Normally we would recommend to put as much business logic as possible on the server-side, and keep the presentation client-side. But it does not work if you want to target stable series. And, after all, this can be seen as a kind of extension of the crash manager.